appinfo/info.xml

```diff
@@ -1,9 +1,15 @@
 <?xml version="1.0"?>
 <info>
 	<id>brute_force_protection</id>
-	<name>Brute Force Protection</name>
-	<description>ownCloud brute force protection app. It hardens ownCloud security by implementing brute force protection.</description>
-	<summary>ownCloud brute force protection app</summary>
+	<name>Brute-Force Protection</name>
+	<description>
+A brute-force attack consists of an attacker trying many user passwords with the hope of eventually guessing correctly and gaining access to a user account thereby. Frequently such attacks are conducted in an automated way which allows attackers to try many different combinations in a short time frame. In addition attackers may use dictionaries to reduce the amount of potential combinations to actual words ("dictionary attack"). Depending on the strength of a password you can estimate the amount of attempts and the respective time it will take to break it and gain access to user data. Administrators should be aware of this potential threat and take respective counter-measures.

```

Comment:
``thereby`` could be deleted. I just sounds like a superfluous word.

``take respective counter-measures``
maybe:
``take appropriate counter-measures``